lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -134,26 +133,6 @@ class TestAdminAccessCoachDashboard(CcxTestCase, LoginEnrollmentTestCase):
     """
     MODULESTORE = TEST_DATA_SPLIT_MODULESTORE
 
-    def make_staff(self):
-        """
-        create staff user
-        """
-        staff = AdminFactory.create(password="test")
-        role = CourseStaffRole(self.course.id)
-        role.add_users(staff)
-
-        return staff
-
-    def make_instructor(self):
-        """
-        create staff instructor
-        """
-        instructor = AdminFactory.create(password="test")
-        role = CourseInstructorRole(self.course.id)
-        role.add_users(instructor)
-
-        return instructor
-
```

Comment:
@doctoryes this test suit is inherited from super class ``CcxTestCase``. All common test utility is inside ``CcxTestCase``.  See
 - https://github.com/mitocw/edx-platform/blob/fix/aq/max_enrollments_and_staff/lms/djangoapps/ccx/tests/test_views.py#L130
- https://github.com/mitocw/edx-platform/blob/fix/aq/max_enrollments_and_staff/lms/djangoapps/ccx/tests/utils.py#L98